Java Script/index.js

```diff
@@ -0,0 +1,81 @@
+/*Loop for adding all the numbers bettwen 0 and 1000 with break > 400*/
+let num = 0;
+let sum = 0;
+for (num = 0; num < 1000; num++) {
+    sum += +num;
+    if (sum > 400) break;
+}
+alert("Sum: " + sum);
+
+/*Loops: Show numbers from 0 to 10*/
+for (let currentNumber = 0; currentNumber < 10; currentNumber++) {
+    alert (`Number ${currentNumber}`);
+}
+console.log ("currentNumber: ", currentNumber)
```

Comment:
If you run it, loop is working 👍 
but in the end `console.log` gives error....
Could you tell why?

Reply:
I redefined it outside of the For loop. It runs in the console now, no "undefined variable" anymore.
Probably the variable needs to be declared first I guess? In the console, they could not find it before executing the loop.
Please let me know if it's the case :)

Reply:
Yeah, so you remember one of those scope "golden" rules.
Variables declared outside are visible inside.
BUT
Variables declared inside are NOT visible outside.
That's what's happening here.
`for(let currentNumber=0....){`   - Means variable declared **inside** loop.
`}`
`console.log(currentNumber)` - And then you have console.log **outside** of loop